libc/include/llvm-libc-macros/linux/sys-epoll-macros.h

```diff
@@ -0,0 +1,38 @@
+//===-- Macros defined in sys/epoll.h header file -------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef LLVM_LIBC_MACROS_LINUX_SYS_EPOLL_MACROS_H
+#define LLVM_LIBC_MACROS_LINUX_SYS_EPOLL_MACROS_H
+
+#include "llvm-libc-macros/linux/fcntl-macros.h"
+
+// TODO: Do we need to define these?
```

Comment:
`<linux/eventpoll.h>`?

Reply:
That's glibc's definition, which indeed seems subtly different from the kernel's (as is noted in that man page).  I think alluding to this issue in this comment would be better than the todo.

Reply:
It sounds like `epoll_event` is packed on x86_64. Our definition in libc/include/llvm-libc-types/struct_epoll_event.h is missing that.